recipes/georaffer/recipe.yaml

```diff
@@ -0,0 +1,54 @@
+context:
+  version: "0.1.0"
+  python_min: "3.10"
```

Comment:
Please remove this `python_min` line. If the value is equal to or less than the current global conda-forge pin, it is unnecessary.